config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_computetargettcpproxies.compute.cnrm.cloud.google.com.yaml

```diff
@@ -106,6 +111,7 @@ spec:
                 type: string
             required:
             - backendServiceRef
+            - location
```

Comment:
It looks like you also added a required field to an existing CRD. This is a breaking change. If we plan to support regional Compute Target TCP Proxy into the existing ComputeTargetTCPProxy CRD, then we need to make this field optional and default it with value `global`.